### PR TITLE
Tweak devotion and clerics system

### PR DESCRIPTION
**Pull request by Animusin**
## About The Pull Request

This PR changes the devotion system. 
Clerical spells are tied to the level of holy skill. The experience is gained for spell casts.
The maximum devotion is 100 and increases by 50 for each level of clerical power.
All clerical spells now consume devotion as intended.
Devotion is gained from prayers. Having the Pantheon cross or neck cross lying nearby increases the amount gained.

Noc cleric gained invisibility and time-stop spells.
Eora cleric gained the mind swap spell.
Templars gained spells (discussable).

Major priests can be not only of Astrata. They have all spells of their domain from the start.

Removed Pestra from available patrons for clerics.

## Why It's Good For The Game

It's good for the game since it discourages being afk for getting all clerical spells. It adds some amount of interactability for clerics, as they need to have a cross nearby to gain devotion. And they can't cast their spells infinitely in a short amount of time.
Having major clerics of all domains makes the game more diverse.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/spells/roguetown/acolyte/eora.dm [hunk 1]
```diff
@@ -48,5 +49,22 @@
...
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+/obj/effect/proc_holder/spell/targeted/mind_transfer/rogue
+	name = "Mind Transfer"
+	desc = ""
+	charge_max = 60 SECONDS
+	clothes_req = FALSE
+	invocation = "Let Eoar show you the world differently!"
```
Comment (sanshoom) [inline code/modules/spells/roguetown/acolyte/eora.dm:61]: should be Eora lol

File: code/datums/gods/patrons/divine_pantheon.dm [hunk 2]
```diff
@@ -102,4 +103,4 @@
 	t0 = /obj/effect/proc_holder/spell/invoked/lesser_heal
 	t1 = /obj/effect/proc_holder/spell/invoked/bud
 	t2 = /obj/effect/proc_holder/spell/invoked/eoracurse
-	t3 = null
\ No newline at end of file
+	t3 = /obj/effect/proc_holder/spell/targeted/mind_transfer/rogue
```
Comment (ThePainkiller) [inline code/datums/gods/patrons/divine_pantheon.dm:106]: Still a bit iffy about the mind transfer spell, people **will** be salty about it. I'd say leave T3 blank until we can think of a proper, unique Eora spell that isn't a body swap

Comment (Optimism333) [reply code/datums/gods/patrons/divine_pantheon.dm:106]: I agree, I do not like the idea of a mind transfer spell.

**Comment by ThePainkiller** [comment]
As spoken during the admin meeting on 08/17, we've decided that both a Mind Swap spell and a Time Stop spell would be too imbalanced and bring about much grief for players, so we're requesting those spells changed with alternatives.
The ones pitched have been:
- Replace Timestop for Noc clerics with a temporal blindness spell **(Suggested by Guayo)**
- Replace mindswap with any other spell for Eorans that would make sense, and is less likely to include skill swap conflicts on use. As #471 is likely to be merged along with this PR, and introducing the Eoran Bud and Wine Bubble spells for T1 and T2 (With modifications pending for those as well), we're gonna need a different, more balanced and less game-experience-ruining spell for Eoran clerics that isn't a mindswap, neither temporal nor permanent.

These two points aside, the PR is good to go, but I'll be waiting until these two points are adressed to give the ok.

**Comment by BadAtThisGame302** [comment]
> As spoken during the admin meeting on 08/17, we've decided that both a Mind Swap spell and a Time Stop spell would be too imbalanced and bring about much grief for players, so we're requesting those spells changed with alternatives.
> 
> The ones pitched have been:
> 
> - Replace Timestop for Noc clerics with a temporal blindness spell **(Suggested by Guayo)**
> 
> - Replace mindswap with any other spell for Eorans that would make sense, and is less likely to include skill swap conflicts on use. As #471 is likely to be merged along with this PR, and introducing the Eoran Bud and Wine Bubble spells for T1 and T2 (With modifications pending for those as well), we're gonna need a different, more balanced and less game-experience-ruining spell for Eoran clerics that isn't a mindswap, neither temporal nor permanent.
> 
> 
> 
> These two points aside, the PR is good to go, but I'll be waiting until these two points are adressed to give the ok.

Noc Acolytes currently only have temporary blindness. Can't have two of those

**Comment by IliaBreev** [comment]
It's not a problem to remove both mind swap and time stop spells, but for now, this Pr is conflicting with https://github.com/Darkrp-community/OpenKeep/pull/471. So it can't really be merged, I left it open only to give an example of how I did it, and how it could be tweaked in FATHENING, since it's likely to be merged.

**Comment by Animusin** [comment]
This PR is conflicting in the devotion system with https://github.com/Darkrp-community/OpenKeep/pull/471, so removing two spells won't help much. I left it open only as an example for the author of the https://github.com/Darkrp-community/OpenKeep/pull/471.

**Pull request closed without merging**